pkg/apis/radix/v1/radixapptypes.go

```diff
@@ -945,45 +945,50 @@ type RadixVolumeMount struct {
 	Path string `json:"path"` // Path within the pod (replica), where the volume mount has been mounted to
 
 	// GID defines the group ID (number) which will be set as owner of the mounted volume.
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
```

Comment:
```suggestion
	// Deprecated: use BlobFuse2 instead.
	// GID defines the group ID (number) which will be set as owner of the mounted volume.
	// +optional
```
The first comment must start with "Deprecated: " for field to be flagged as deprecated

Reply:
Fixed